src/renderer/components/Visualiser/TopBar/GraqlEditor/GraqlEditor.vue

```diff
@@ -253,10 +252,16 @@ export default {
 
       this.initialEditorHeight = $('.CodeMirror').height();
 
+      // for some reason, CodeMirror adds an anonymous <div> inside the auto generated .CodeMirror-scroll element.
+      // this <div> has a height of 50px (!) which distorts the GraqlEditor by allowing scroll when it shouldn't
+      // we're setting this height of this seeminly useless <div> to 0
+      document.getElementsByClassName('CodeMirror-scroll')[0].children[1].style.height = 0;
```

Comment:
eh? is this a hack :P

Reply:
oh yeah! couldn't be more of it :D

Reply:
@flyingsilverfin in https://github.com/graknlabs/workbase/pull/308/commits/d9f4a15b60ddbd4ff51274fb80c04201f9d37119 I have moved this hack from `js` to `css` (which imo is more toleratable)